tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,10 +125,65 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_auth_email_checks(self):
+        # Setting invalid user.email
+        # newgitdir = mkdtemp()
+        # os.chdir(newgitdir)
+        # self.run_git_command('init')
+        self.run_git_command('config', 'user.email invalid#!format!*!@email.com')
```

Comment:
Line is longer than allowed. (81 > 79)

*LineLengthBear, severity NORMAL, section `linelength`.*